examples/UWPCE_Python/git_push_test_class1

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
you shouldn't be adding anything to the examples dir.

you can remove this with:

git rm examples/UWPCE_Python/git_push_test_class1

then:

git commit -a

git push

and this PR will be updated.